documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -128,6 +131,82 @@ For a new model that only has an Aspect so far:
 
 image::different-constraints.png[Different Constraints]
 
+[[abstract-entity-case]]
+=== Use Abstract Entities
+
+There are two different ways to create an abstract entity:
+
+* Use drag-and-drop from the Workspace sidebar
+* By adjusting an Entity in the extends field
+
+*Use drag-and-drop from the Workspace sidebar To add abstract entity, proceed as follows:*
+
+. Open the Workspace sidebar
+. Drag the Abstract Entity element and drop it to the graph area canvas
+
+image::drag-and-drop-abstract-entity.png[Drag and drop Abstract Entity]
+
+*Adjusting an Entity in the extends field*
+
+To select or create an abstract entity in the Edit field of an Entity, proceed as follows:
+
+. Double-click the entity or abstract entity that should derive from another entity.
+. Abstract entity can be created or selected in the "extends" field.
+
+image::extends-field-abstract-entity.png[Extends fiels Abstract Entity]
+
+[[abstract-entity-example]]
+=== Abstract Entities Example
+
+For a new model that only has an Aspect so far:
+
+. Create one Property by clicking twice on the Aspect's Add icon image:add.png[Add icon].
+. Create one Characteristic by clicking the Add icon image:add.png[Add icon].
+. Create one Entity by clicking the bottom Add icon image:add.png[Add icon].
+. Drag and Drop an Abstract Entity from the sidebar.
+. Connect the Abstract Entity with the Entity:
+.. Click the Entity,
+.. Then CTRL+click the Abstract Entity,
+.. Then click the Connect icon image:toolbar-icons/connect.png[Connect icon] on the xref:ui-overview.adoc#toolbar[toolbar].
```

Comment:
Then CTRL+click the Abstract Entity and click the Connect icon image:toolbar-icons/connect.png[Connect icon] on the xref:ui-overview.adoc#toolbar[toolbar].